src/helpers/simplify_claims.ts

```diff
@@ -35,7 +35,8 @@ function simplifyPropertySnaks (propertySnaks, options) {
     .map(claim => simplifyClaim(claim, options))
     // Filter-out novalue and somevalue claims,
     // unless a novalueValue or a somevalueValue is passed in options
-    .filter(isDefined)
+    // Considers null as defined
+    .filter(obj => obj !== undefined)
```

Comment:
This change is a matter of taste, right? or is there a benefit here I'm not seeing?

Reply:
yes and no.

When it's inside the `.filter()` or `.map()` then typescript can assume types better without adding too many types manually. At least that's my personal experience.

When its used often and types are added then its mainly a matter of taste or just simpler to write a method once and use it often.

In this case it was a bit easier for me to understand what's happening when reading the implementation rather than the method name. So yeah, personal taste was involved.